src/cli.ts

```diff
@@ -118,6 +118,10 @@ if (require.main === module) {
       '-e, --electron-version <value>',
       "electron version to package, without the 'v', see https://github.com/electron/electron/releases",
     )
+    .option(
+      '--wvvmp',
+      'use widevine enabled beta version of electron',
+    )
```

Comment:
Please document the new flag in `docs/api.md`

Reply:
How does the following sound?
```ts
#### [wvvmp]

'''
--wvvmp
'''

Widevine-enabled version of Electron, see https://github.com/castlabs/electron-releases.
```

Reply:
Looks good and follows the current structure.

Nit: make the description start with a verb? I'd prefix your description with "Use a "

Reply:
:thumbsup: